x-pack/solutions/security/plugins/session_view/public/components/detail_panel_alert_actions/styles.ts

```diff
@@ -1,107 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { useMemo } from 'react';
-import { useEuiTheme, transparentize } from '@elastic/eui';
-import { CSSObject, css } from '@emotion/react';
-
-interface StylesDeps {
-  minimal?: boolean;
-  isInvestigated?: boolean;
-}
-
-export const useStyles = ({ minimal = false, isInvestigated = false }: StylesDeps) => {
```

Comment:
these styles were not used. Removing so that we don't need to think about color palette